source/draft_api.rst

```diff
@@ -396,43 +400,18 @@ GET /projects/:slug?include_revisions=true
       "deleted_at": null,
       "updated_at": "2015-04-17",
       "users": {
-        "members": [
-          "patcht",
-          "tschuy"
-        ],
-        "spectators": [
-          "tschuy"
-        ],
-        "managers": [
-          "tschuy"
-        ]
-      },
-      "parents":
-      [
-        {
-          "uri": "https://code.osuosl.org/projects/ganeti-webmgr",
-          "name": "Ganeti Web Manager",
-          "uuid": "a034806c-00db-4fe1-8de8-514575f31bfb",
-          "revision": 3,
-          "created_at": "2015-04-16",
-          "deleted_at": null,
-          "updated_at": null,
-          "users": {
-            "members": [
-              "patcht",
-              "tschuy"
-            ],
-            "spectators": [
-              "tschuy"
-            ],
-            "managers": [
-              "tschuy"
-            ]
-          }
+        "user1": {
+          "member": true,
+          "spectator": false,
+          "manager": false
+        },
+        "user2": {
+          "member": true,
+          "spectator": true,
+          "manager": true
         },
```

Comment:
Any reason you removed the ``parents`` field from this example? It's supposed to be an example of the ``parents`` field in use.